lib/airbrake-ruby/notifier.rb

```diff
@@ -120,6 +121,21 @@ def notify_request(request_info)
       @route_sender.notify_request(request_info, promise)
     end
 
+    # @macro see_public_api_method
+    def notify_sql(sql_info)
```

Comment:
In gobrake we ended with:
- notifier.Routes.Notify(route_info), notifier.Routes.AddFilter
- notifier.Queries.Notify(query_info), notifier.Queries.AddFilter

Perhaps you can do something similar here...

Reply:
Yeah, I also noticed that we can decouple this from the `Airbrake` module. I'll think about the design after this PR is merged, so it doesn't distract me from the ultimate goal of sending SQL.